bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs

```diff
@@ -51,6 +53,46 @@ public void Test_Encrypt_Ballot_Simple_Succeeds()
             Assert.That(ciphertext.ObjectId == submitted.ObjectId);
         }
 
+        [Test]
+        public void Test_Encrypt_Ballot_Undervote_Succeeds()
+        {
+            // Configure the election context
+            var keypair = ElGamalKeyPair.FromSecret(Constants.TWO_MOD_Q);
+            var manifest = ManifestGenerator.GetJeffersonCountyManifest_Minimal();
+            var internalManifest = new InternalManifest(manifest);
+            var context = new CiphertextElectionContext(
+                1UL, 1UL, keypair.PublicKey, Constants.TWO_MOD_Q, internalManifest.ManifestHash);
+            var device = new EncryptionDevice(12345UL, 23456UL, 34567UL, "Location");
+            var mediator = new EncryptionMediator(internalManifest, context, device);
```

Comment:
That's a lot of copy and pasting between the setup of these two tests.  I personally prefer in these cases to use helper methods with descriptive names to help make tests human readable ala Clean Code, almost like making a DSL per test file.  That's because I prefer to view tests as documentation first that's executable and verifiable second.  FWIW I've read over these tests a few times and I don't understand what they're saying.  So, if getting the tests understandable by someone with my minimal level of understanding is the objective, then they fail.  But that can be a lot of work, and with a looming deadline it may not be worth it.

Reply:
https://github.com/microsoft/electionguard-cpp/issues/229